src/vorta/views/main_window.py

```diff
@@ -156,19 +167,17 @@ def profile_add_action(self):
             self.profileSelector.setCurrentIndex(self.profileSelector.currentIndex())
 
     def backup_started_event(self):
-        self.set_status(progress_max=0)
         self._toggle_buttons(create_enabled=False)
+        self.set_log('')
 
     def backup_finished_event(self):
-        self.set_status(progress_max=100)
         self._toggle_buttons(create_enabled=True)
         self.archiveTab.populate_from_profile()
         self.repoTab.init_repo_stats()
 
     def backup_cancelled_event(self):
         self._toggle_buttons(create_enabled=True)
-        self.set_status(progress_max=100)
-        self.set_status(self.tr('Task cancelled'))
+        self.set_log(self.tr('Task cancelled'))
 
```

Comment:
```suggestion
        font = self.progressText.font()
        font.setBold(False)
        self.progressText.setFont(font)

```

Reply:
Fully removed the bold font. Since we update the progress very often, we should avoid extra instructions.